k8s/Makefile

```diff
@@ -40,6 +40,10 @@ export APP_SECURE_HSTS_SECONDS = 31536000
 export APP_AUTOMATICALLY_INGEST_CONTENT ?= True
 export APP_NOTIFY_AFTER_INGESTING_CONTENT ?= False
 
+# Task-completion survey config
+export APP_TASK_COMPLETION_SURVEY_URL ?= undefined
+export APP_TASK_COMPLETION_SURVEY_PERCENTAGE ?= 5.00  # default 5%
+
```

Comment:
@limed can I get your 👍 on the various k8s changes made in this PR, please?

Reply:
This looks good over here